src/test/store/receive-profile.test.js

```diff
@@ -311,7 +352,8 @@ describe('actions/receive-profile', function() {
       headers: {
         get: () => 'application/json',
       },
-      json: () => Promise.resolve(_getSimpleProfile()),
+      json: () =>
+        Promise.resolve(JSON.parse(serializeProfile(_getSimpleProfile()))),
```

Comment:
You'll see this in other places in this file: we were incorrectly returning a processedProfile, and not a serializedProfile. So I fixed that in the whole file, but this makes it somewhat more verbose. Also you'll see there are different ways of returning a fetch response in this file. I especially like the one I implemented for the "compare" feature and I may change it in a future patch, but I felt it was out of the scope for this already large PR.